breeze-for-qubes/breeze-for-qubes.spec

```diff
@@ -0,0 +1,309 @@
+%if 0%{?qubes_builder}
+%define _sourcedir %(pwd)/breeze-for-qubes
+%define _builddir %(pwd)/breeze-for-qubes
+%define breeze_epoch %(cat epoch_breeze)
+%define breeze_version %(cat version_breeze)
+%else
+%define breeze_epoch 0
+%define breeze_version 5.5.4
+%endif
+
+%global         base_name   breeze
+
+%global         build_kde4  1
+
+Name:    plasma-breeze
```

Comment:
I would name the package differently (`plasma-breeze-qubes`?), then add appropriate dependencies like `Conflicts: plasma-breeze`, or even `Provides: plasma-breeze`. If possible, that would be better than setting epoch, which is really hard to rollback in the future if needed.